doc/scrapper/ParsingFiles.py

```diff
@@ -0,0 +1,350 @@
+    string = string.split(',')
+    return string
+def parse_title(string):
+    if(string == None):
+        return string
+    return string[1:-2]
+def parse_doi(string):
+    if(string == None):
+        return string
+    return string[1:-2]
+def parse_sources(string):
+    if(string == None):
+        return string
+    string = string[1:-2]
+    string = string.replace('"','')
+    string = string.split(',')
+    return string
+def parse_doiUrl(string):
+    if(string == None):
+        return string
+    return string[1:-2]
+
+def parce_venue(string):
+    if(string==None):
+        return string
+    string = string[1:-3]
+    return string
+    
+    
+    
+
+
+# In[10]:
+
+
+df['journalVolume'] = df['journalVolume'].apply(parse_journalVolume)
+df['journalPages'] = df['journalPages'].apply(parse_journalPages)
+df['pmid'] = df['pmid'].apply(parse_pmid)
+df['year'] = df['year'].apply(parse_year)
+df['outCitations'] = df['outCitations'].apply(parse_outCitations)
+df['s2Url'] = df['s2Url'].apply(parse_s2Url)
+df['s2PdfUrl'] = df['s2PdfUrl'].apply(parse_s2PdfUrl)
+df['id'] = df['id'].apply(parse_id)
+df['authors'] = df['authors'].apply(parse_authors)
+df['journalName'] = df['journalName'].apply(parse_journalName)
+df['paperAbstract'] = df['paperAbstract'].apply(parse_paperAbstract)
+df['inCitations'] = df['inCitations'].apply(parse_incitations)
+df['pdfUrls'] = df['pdfUrls'].apply(parse_pdfUrls)
+df['title'] = df['title'].apply(parse_title)
+df['doi'] = df['doi'].apply(parse_doi)
+df['sources'] = df['sources'].apply(parse_sources)
+df['doiUrl'] = df['doiUrl'].apply(parse_doiUrl)
+df['venue'] = df['venue'].apply(parce_venue)
+                                
+#just to being possible to wokr with stuff change None to -1
+df.fillna(-1, inplace=True)
+
+
+# In[11]:
+
+
+#creating some usefull information
+def getNumcitations(x):
+    if(x==-1):
+        return 0
+    return len(x)
+
+df['NumOutCitations'] = df['outCitations'].apply(getNumcitations)
+df['NumInCitations'] = df['inCitations'].apply(getNumcitations)
+
+
+# In[12]:
+
+
+#No null df
+Cyteddf = df[(df['NumInCitations'] != 0) & (df['NumOutCitations'] != 0) ]
+del df
+
+
+# In[13]:
+
+
+len(Cyteddf)
+
+
+# In[14]:
+
+
+Cyteddf.sort_values(by = 'NumInCitations',ascending=False)
+Cyteddf = Cyteddf.reset_index()
+
+
+# In[16]:
+
+
+inCitdic = {}
+outCitdic = {}
+related_ind = []
+for ind in Cyteddf.index:
+    for Cit in Cyteddf.loc[ind]['inCitations']:
+        inCitdic[Cit] = 1 
+        
+    for Cit in Cyteddf.loc[ind]['outCitations']:
+        outCitdic[Cit] = 1 
+        
+for index,idis in enumerate(Cyteddf['id']):
+        if (idis in inCitdic) or (idis in outCitdic):
+            related_ind.append(index)
+
+
+# In[19]:
+
+
+related_papers = Cyteddf.loc[related_ind]
+
+
+# In[22]:
+
+
+related_papers = related_papers.drop('index', 1)
+related_papers
+
+
+# In[23]:
+
+
+related_papers.to_csv("/home/ricostynha/Desktop/first1000entries",sep = "|",na_rep = "* *",index = False)
```

Comment:
Can you please change this path to `./first1000entries`? This will put the first 1000 entries directly into folder `doc/scrapper`, independently of your local machine.